src/modules/formBuilder/components/FormBuilder.tsx

```diff
@@ -122,100 +211,9 @@ const FormBuilder: React.FC<FormBuilderProps> = ({
               enableBehavior: EnableBehavior.Any,
             };
 
-            const newDefinition: FormDefinitionJson = {
-              ...workingDefinition,
-              item: [...workingDefinition.item, newItem],
-            };
-
-            setWorkingDefinition(newDefinition);
+            setSelectedItem(newItem);
```

Comment:
Okay, one more thread to pick up here. Should the user be able to create a new item and immediately save it, without changing any of the item's attributes? Right now this isn't possible, because the new item's form state is clean. 

It might be a confusing UX if you can't find the save button. But on the other hand, maybe it's a good thing to require changing those defaults before save.

Reply:
Hmm, I think that's fine to require them to change some values before submitting.  If that become an issue we can revisit